package.json

```diff
@@ -1,7 +1,23 @@
 {
   "name": "app",
   "private": "true",
+  "dependencies": {
```

Comment:
- [ ] It would be better if we add the `babel-loader` and `react-dom` packages in the package.json file
![Screenshot from 2023-08-07 08-09-24](https://github.com/SabaAhmad404/Hello-rails-react/assets/46388113/c5fceade-3d16-4fe6-bae6-00cac3cb7cfc)
As we can see in the screenshot above, the missing babel loader causes the error
![Screenshot from 2023-08-07 08-11-58](https://github.com/SabaAhmad404/Hello-rails-react/assets/46388113/16d17b9f-d13b-4ed1-a63d-df6861af95f2)